astropy/visualization/tests/test_time.py

```diff
@@ -195,7 +195,7 @@ def test_formats(format, expected):
                 Time("2077-03-22T12:30:32.1", scale=DEFAULT_SCALE),
             )
         assert get_ticklabels(ax.xaxis) == expected
-        ax.get_xlabel() == f"Time ({format})"
+        assert ax.get_xlabel() == f"Time ({format})"
```

Comment:
I think what is in brackets should be the scale not the format:

https://github.com/astropy/astropy/blob/b6006cd9b83cb6197234ca3e78efaa3109369094/astropy/visualization/time.py#L246

So this is a bug in the test, it should always be ``Time (tai)`` I think in this case.

Reply:
This pull request has stalled, so consider fixing this yourself.